package.json

```diff
@@ -55,6 +57,15 @@
     "@jobber/generators": "file:packages/generators",
     "@jobber/stylelint-config": "file:packages/stylelint-config",
     "@reach/router": "^1.3.3",
+    "@storybook/addon-actions": "^6.5.16",
```

Comment:
Is there a reason we aren't just starting with `7.0` since we're creating a fresh instance?

Reply:
I used `storybook init` on this one and that's what it installed. haha. Probably because 7 is still on beta?

Reply:
Ah I see `6.5.16` is the "latest" release indeed. The way they have been promoting 7 I assumed it was full release.

Reply:
looks pretty straightforward to go there. lemme try it. 

Yeah like this one https://nx.dev/packages/storybook/documents/storybook-7-setup It's ready!! But it's actually not!!

Reply:
lil update on this. We need to be on node 16 to use 7 😢